.github/workflows/run-tests.yml

```diff
@@ -487,12 +488,24 @@ jobs:
           annotate_only: true
           skip_annotations: true
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -e          
+          job_id=$(curl -L \
```

Comment:
I'd add a `--fail` too.

Reply:
add to what?

Reply:
I add it, and then remove it. For some reason it start failing with the changes you proposed. It is not clear what exactly was causing it to fail, so I rollback to a working solution.

Reply:
Hmm.. I wouldn't expect this to be an issue. Can you check why `--fail` is failing? It's generally good practice to have this set since it ensures you get a 2xx response.